src/app/core/common/mist-effects.ts

```diff
@@ -1,5 +1,5 @@
 import { Injectable } from '@angular/core';
-import { Http, Headers } from '@angular/http';
+import { HttpClient, HttpHeaders, HttpResponseBase } from '@angular/common/http';
```

Comment:
Remove unnecessary HttpResponseBase. Make sure everything is working OK.

Reply:
done